src/ClientApp/src/styles/main.scss

```diff
@@ -1,10 +1,120 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:100,300,400");
+
+$font-primary: 'Roboto', sans-serif;
+$font-size: 200;
+$primary: #2196F3;
+$secondary: #90CAF9;
+$ternary: #BBDEFB;
+$primary-dark: #0D47A1;
+$background: #FFF;
+$background-invert: darken($background, 50%);
+
+$navigation-span: 160px;
+
 * {
     margin: 0 0;
 }
 
+body {
+    font-family: $font-primary;
+    font-size: $font-size;
+    background: $background;
+    color: $primary;
+}
+
+.logo {
+    width: 120px;
+    height: 49px;
+    border: 0;
+    padding: 5px;
+    vertical-align: middle;
+    margin-left: -$navigation-span;
+}
+
+header {
+    text-align: center;
+    width: 100%;
+}
+
+header span {
+    padding: 5px $navigation-span 5px 5px;
+}
+
 footer {
     position: absolute;
-    bottom: 0;
+    bottom: 5px;
     width: 100%;
     text-align: center;
+}
+
+nav {
+    overflow: hidden;
+    background-color: $ternary;
+    display: inline;
+    padding: 10px;
+    border-radius: 10px;
+}
+
+nav a {
+    color: $primary;
+    text-align: center;
+    padding: 10px;
+    text-decoration: none;
+}
+
+nav a:hover, .dropdown:hover .dropbtn {
+    background-color: $secondary;
+    color: $background-invert;
+}
+
+.menu-space {
+    display: inline-block;
+    width: 100px;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: $ternary;
+    min-width: 160px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+    margin-top: 10px;
+    border-radius: 10px;
+}
+
+.dropdown-content a {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Expected selector ".dropdown-content a" to come before selector "nav a:hover" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=css%3AS4664)